src/app/entry/current-collections/current-collections.component.html

```diff
@@ -18,21 +18,30 @@
     <mat-card-title>Collections</mat-card-title>
   </mat-card-header>
   <mat-card-content>
-    <mat-list>
-      <mat-list-item
-        [matTooltip]="collectionOrganizations.organizationDisplayName"
-        *ngFor="let collectionOrganizations of currentCollections$ | async"
-      >
-        <a
-          [routerLink]="['/organizations', collectionOrganizations.organizationName, 'collections', collectionOrganizations.collectionName]"
-          routerLinkActive="router-link-active"
-          >{{ collectionOrganizations.collectionDisplayName }}</a
-        >
-      </mat-list-item>
-    </mat-list>
-    <div class="p-3">
+    <ng-container *ngFor="let collectionOrganization of currentCollections$ | async">
+      <div class="mb-1 single-spaced">
+        <ng-container *ngIf="collectionOrganization.organizationAvatarUrl; then withAvatar else withoutAvatar"></ng-container>
+        <ng-template #withAvatar>
+          <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="1rem">
+            <img class="tiny-org-logo" [src]="collectionOrganization.organizationAvatarUrl" alt="org logo" [appFallback]="orgLogoService.DEFAULT_URL"/>
```

Comment:
Does this mean the logic is:

1. If valid avatar, display it.
2. If invalid avatar, display generic logo
3. If no avatar, display text.

If I'm right,  I would lean towards "If invalid or no avatar, display text". But I don't feel super strongly about this.

Reply:
Also, I think the `alt` text should include the organization name, e.g., `Org logo for <org>`

Reply:
> Does this mean the logic is:
> 
> 1. If valid avatar, display it.
> 2. If invalid avatar, display generic logo
> 3. If no avatar, display text.
> 
> If I'm right, I would lean towards "If invalid or no avatar, display text". But I don't feel super strongly about this.

Yes, that's the way it works.  I'd recommend we leave it as is, because:
1. The behavior is consistent with the way the org logo works everywhere else on the site.
2. The code is much more simple and easy to understand.

Reply:
> Also, I think the `alt` text should include the organization name, e.g., `Org logo for <org>`

done.